.github/workflows/ci.yml

```
@@ -203,3 +203,35 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           fail_ci_if_error: true
+  generate-locales:
```

Please add a blank line here

```suggestion

  generate-locales:
```

---

Sorry for my simple mistake, I fixed it.